modules/libs/entities/school.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne, JoinColumn } from 'typeorm';
+import { Organization } from './organization'; // Assuming you have an Organization entity
+
+@Entity({ name: 'schools' })
+export class School {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  name: string;
+
+  @Column()
+  organizationId: string;
```

Comment:
It shouldn't be a NULLABLE field. The school MUST belong to an organization.

```suggestion
  @Column({ nullable: false })
  organizationId!: string;
```